src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -87,7 +88,23 @@ private set
         private Uri link;
         private string graphPath;
         private string content;
-        private string name;
+        private string packageName;
+
+
+        /// <summary>
+        /// Package Name
+        /// </summary>
+        internal string PackageName
```

Comment:
it still feels pretty odd to me that the DocumentationBrowserViewModel has a single piece of state like this that it has to hold on to.

If I am understanding correctly, can we at least name it something like:
`CurrentPackageName` or something like that and add a comment describing when it will be null and when it will be valid?

Reply:
@mjkkirschner notice that the dependency on PackageName was added in the next PR:
https://github.com/DynamoDS/Dynamo/pull/13414
Renamed to CurrentPackageName in the next commit: https://github.com/DynamoDS/Dynamo/pull/13703/commits/f00db5b919450b0ccdafac4c9f90a8827b5d80c1
![image](https://user-images.githubusercontent.com/61755417/216129140-c5d1f11d-1376-4aad-b901-60ce3a393f5c.png)